db/seeds/categories.rb

```diff
@@ -1,3 +1,5 @@
-puts Category.create(name: "T-shirts", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/mens_t-shirt--600x600.jpg")
-puts Category.create(name: "Household Items", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/mug_with_MUG_joke--600x600.jpg")
-puts Category.create(name: "Stationery", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/notepad--600x600.jpg")
+puts Category.create(name: "Décor", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/newimages/category_decor_600.jpg")
+puts Category.create(name: "Food Prep", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/newimages/category-food_prep_600.jpg")
+puts Category.create(name: "Household Goods", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/newimages/category-houseware_600.jpg")
+puts Category.create(name: "Stationery", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/newimages/category_stationery_600.jpg")
+puts Category.create(name: "T-shirts", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/newimages/category_t-shirts_600.jpg")
```

Comment:
Line is too long. [139/80]<br>Do not write to stdout. Use Rails' logger if you want to log.